src/app/code/community/Hackathon/PromoCodeMessages/Model/Validator.php

```diff
@@ -462,6 +466,25 @@ protected function _formatMessage($message, $params = '', $internalMessage = nul
         return $message;
     }
 
+    /**
+     * Wrap the message in a li
+     *
+     * @param string $message
+     * @param string $params
+     * @param string $internalMessage
+     *
+     * @return string containing entire error message
+     */
+    protected function _formatMessageWithContainer($message, $params = '', $internalMessage = null)
+    {
+        if (!$message) {
+            return '';
+        }
+        $message = '<li class="promo_error_item">' . $this->_helper->__($message, $params) . '</li>';
```

Comment:
No, because the nested messages come with container already back. But we have the problem, that we try to translate the already nested stuff. So in theory we should refactor all that, so that we have arrays with the messages, then we translate everything and then we format it with html. But I don't have the right mind currently to do that properly and test it.